clients/servicenowclient/client.go

```diff
@@ -0,0 +1,48 @@
+package servicenowclient
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/mattermost/mattermost-app-servicenow/app"
+	"github.com/mattermost/mattermost-app-servicenow/config"
+	"github.com/mattermost/mattermost-app-servicenow/store"
+)
+
+type Client struct {
+	client *http.Client
+}
+
+func NewClient(userID string) *Client {
+	ctx := context.Background()
+	conf := app.GetOAuthConfig()
+	token, found := store.GetToken(userID)
+	if !found {
+		return nil
+	}
+	return &Client{
+		client: conf.Client(ctx, token),
+	}
+}
+
+func (c *Client) CreateIncident(table string, v interface{}) error {
+	url := fmt.Sprintf("%s/%s/%s", config.ServiceNowInstance(), "/api/now/table/", table)
```

Comment:
I think these slashes will cause extra slashes to be put on the URL

```suggestion
	url := fmt.Sprintf("%s/%s/%s", config.ServiceNowInstance(), "api/now/table", table)
```